client/src/pages/privacy.md

```diff
@@ -0,0 +1,216 @@
+
+We will ask you to verify your identity before we give you access to your information or correct it, and we will try to make the process as simple as possible. If we refuse to give you access to, or correct, your personal information, we must notify you in writing setting out the reasons. 
+
+If we make a correction and we have disclosed the incorrect information to others, you can ask us to tell them about the correction. We must do so unless there is a valid reason not to. 
+
+If we refuse to correct your personal information, you can ask us to associate with it (for example, attach or link) a statement that you believe the information is incorrect and why. 
+
+You also have the right under the Freedom of Information Act 1982 to request access to documents that we hold and ask for information that we hold about you to be changed or annotated if it is incomplete, incorrect, out-of- date or misleading.   
+
+## How to contact us 
+
+Requests for access to your own personal information are free. 
+
+Request for your own personal information may be made to: 
+
+**Email** 
+<a href="mailto:foi@dta.gov.au">foi@dta.gov.au</a>
+   
+**Post** 
+FOI Officer 
+Digital Transformation Agency 
+PO Box 457 
+Canberra City 
+ACT 2601 
+
+We will respond to request to access (and where relevant, correct) your own personal information within 30 days of receiving the request.  If we are required or permitted to refuse access, we will give you a written notice setting out: 
+
+* the reasons for refusing access (unless it would be inappropriate and unreasonable to do so having regard to the circumstances); and 
+
+* the mechanisms available to you to complain about the refusal. 
+
+## How to make a complaint 
+
+If you wish to make a complaint about how DTA’s has handled your personal information, please do so in writing.  If you need help lodging a complaint, you can contact the Privacy Officer on 02 6120 8595. 
+
+Complaints can be made to: 
+
+**Email** 
+<a href="mailto:privacy@dta.gov.au">privacy@dta.gov.au</a>
+
+**Post** 
+Privacy Officer 
+Digital Transformation Agency 
+PO Box 457 
+Canberra City 
+ACT 2601 
+
+If we receive a complaint from you, we will acknowledge your complaint within 3 business days of receiving the complaint.   
+
+We will respond to your complaint within 30 days (or another timeframe agreed with you) of receiving your complaint and explain the actions we have taken or propose to take to address the issues raised in your complaint.  If you are not satisfied with DTA’s response to your complaint, you may ask for a review by a senior officer within DTA.  You may also lodge a complaint to the Office of the Australian Information Commissioner by email to <a href="mailto:enquiries@oaic.gov.au">enquiries@oaic.gov.au</a> or by post to GPO Box 5218, Sydney NSW 2001. 
```

Comment:
Do we want to format the address at the end on new lines?

Reply:
Yes please! Great suggestion